src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/CurlyBraceCompletionSession.cs

```diff
@@ -224,6 +224,19 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
                 return base.GetAdjustNewLinesOperation(previousToken, currentToken, optionSet, nextOperation);
             }
 
+            public override void AddAlignTokensOperations(List<AlignTokensOperation> list, SyntaxNode node, OptionSet optionSet, NextAction<AlignTokensOperation> nextOperation)
```

Comment:
But the } already aligns with the { when doing smart indenting too, and why is this needed only here in CurlyBraceSession, not also in the base case?